packages/ia-zendesk-help-widget/src/ia-zendesk-help-widget.js

```diff
@@ -0,0 +1,75 @@
+import { html, LitElement } from 'lit-element';
+import zendeskWidgetCSS from './styles/zendesk-widget';
+import questionIcon from './icon-question-mark';
+
+class ZenDeskHelpWidget extends LitElement {
+  static get styles() {
+    return zendeskWidgetCSS;
+  }
+
+  static get properties() {
+    return {
+      widgetSrc: { type: String },
+      widgetkey: { type: String },
+    };
+  }
+
+  constructor() {
+    super();
+    this.widgetSrc = '';
+    this.widgetkey = '';
+  }
+
+  initiateZenDesk() {
+    // Hide custom button
+    this.style.opacity = 0;
```

Comment:
All styles should be housed within the `styles` template that is returned from the static method and a class should be used to toggle opacity instead. Typically, you would do this by updating a tracked property on the component, like `visible`. Then you can have the default style set to `opacity: 0` and the visible class override it.